modules/abstract-lightning/src/wallet/lightning.ts

```diff
@@ -364,6 +370,32 @@ export class LightningWallet implements ILightningWallet {
       throw new Error(`serialized txHex is missing`);
     }
 
+    const walletData = this.wallet.toJSON();
+    if (!walletData.coinSpecific.watchOnlyAccounts) {
+      throw new Error(`wallet is missing watch only accounts`);
+    }
+
+    const watchOnlyAccountDetails = decodeOrElse(
+      WatchOnly.name,
+      WatchOnly,
+      walletData.coinSpecific.watchOnlyAccounts,
+      (errors) => {
+        throw new Error(`invalid watch only accounts`);
+      }
+    );
+    const network = getUtxolibNetwork(this.wallet.coin());
+
+    try {
+      validatePsbtForWithdraw(
+        transactionRequestCreate.transactions[0].unsignedTx.serializedTxHex,
+        network,
+        params.recipients,
+        watchOnlyAccountDetails.accounts
+      );
+    } catch (err: any) {
+      throw new Error(`error validating withdraw psbt: ${err.message}`);
```

Comment:
you are dropping the stack here, add log the error with the full stack as well here some way